lib/logthrdest/logthrdestdrv.c

```diff
@@ -33,6 +33,24 @@
 
 static void _init_stats_key(LogThreadedDestDriver *self, StatsClusterKey *sc_key);
 
+const gchar *
+logthreadedresult_to_str(LogThreadedResult self)
```

Comment:
can you please use underscores in the name to make this more readable?

Reply:
Fixed.